src/python/pants/backend/python/register.py

```diff
@@ -53,6 +56,7 @@ def register_goals():
   task(name='interpreter', action=SelectInterpreter).install('pyprep')
   task(name='requirements', action=ResolveRequirements).install('pyprep')
   task(name='sources', action=GatherSources).install('pyprep')
+  task(name='python-dists', action=PythonCreateDistributions).install('pyprep')
```

Comment:
Installing this task on pyprep seems like the best place to put it, however with this choice it will only work in `./pants run` if the produced .whl is dumped into the leftmost runner pex, which seems like an anti-pattern. Thoughts?